.jenkins/actions/build_bare_metal.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash
 
-# stop on all errors (also on commands with a pipe-redirection)
+# This script provides a Jenkins action to build bare metal executables
+# on the Piz Daint system at CSCS. Several environment variables (see below)
+# are expected to be set upon execution.
+# If the build runs through successfully, the resulting executables are stored.
+
+# 2021/01/22 Oliver Fuhrer, Vulcan Inc, oliverf@vulcan.com
+
+# stop on all errors (also on errors in a pipe-redirection)
 set -e
 set -o pipefail
 
+# the following environment variables need to be set
+#   CONFIGURATION_LIST - Space-separated list of configurations to test executables with
```

Comment:
might be helpful to specify whether these correspond to the configuration filename or the experiment_name specified in the yaml (looks like it is the latter)

Reply:
Great suggestion!